main.js

```diff
@@ -0,0 +1,156 @@
+import './style/style.sass'
+import './style/reset.css'
+
+const main = document.querySelector('.main')
+
+const array = []
+let selectedCards = []
+let selectedDivCards = []
+let score = 0
+let cardPairs
+
+const ApiKey = '1c851fa8-2663-4d78-a090-0b1ed887373d'
+const urlCat = 'https://api.thecatapi.com/v1/images/search?mime_types=jpg,png&limit=8'
+
+fetch(urlCat, {
+    headers: {
+        'x-api-key': ApiKey
+    }
+})
+    .then(response => response.json())
```

Comment:
you should check if `response.ok`. I told you about it. It will never go to `catch`